charts/opentelemetry-collector/README.md

```diff
@@ -190,6 +190,91 @@ presets:
     extractAllPodAnnotations: true
 ```
 
+### Configuration for Annotation-Based Discovery
+
+The collector can be configured to automatically discover and collect telemetry from pods based on annotations. This feature provides a drop-in replacement for the `logsCollection` preset, allowing for selective collection of logs and metrics from specific pods rather than collecting from all pods in the cluster.
```

Comment:
Only the `presets.annotationDiscovery.logs` can act as drop-in replacement. In this paragraph we can be more generic about annotation based discovery and then provide a specific section later for the relation of `logsCollection` and `annotationDiscovery.logs` presets?

Reply:
Reviewed. Some final suggestions and it should be good. Feel free to resolve this comment and the rest comments of this review :). (I can't do it since I don't see any resolve button)